Notation.Plugin.AzureKeyVault/KeyVault/KeyVaultClient.cs

```diff
@@ -196,13 +196,18 @@ public async Task<X509Certificate2Collection> GetCertificateChainAsync()
                     {
                         // macOS doesn't support non-encrypted MAC
                         // https://github.com/dotnet/runtime/issues/23635
+
+                        // Import will reverse the order of the certificates 
+                        // in the chain
                         chain.Import(
                             rawData: Pkcs12.ReEncode(Convert.FromBase64String(secretValue)),
                             password: null,
                             keyStorageFlags: X509KeyStorageFlags.DefaultKeySet);
                     }
                     else
                     {
+                        // Import will reverse the order of the certificates 
+                        // in the chain
```

Comment:
Both `if` and `else` statements have this comment, thus we can extract it and comment before the `if` statement.

Reply:
Updated.